README.md

```diff
@@ -1,69 +1,58 @@
 # List-KR
 
-힌국어로 AdGuard나 uBlock Origin 또는 List-KR에 대해 아시고 싶으신 점이 있으신가요? [List-KR Wiki에 방문](https://github.com/List-KR/List-KR/wiki)해보세요.
+[List-KR 위키에 방문](https://github.com/List-KR/List-KR/wiki)하여 AdGuard, uBlock Origin, 그리고 List-KR에 대해 더 알아볼 수 있습니다.
 
-AdGuard, uBlock Origin에 대한 질문이나 문제가 있으신가요? [List-KR Discussions에 방문](https://github.com/List-KR/List-KR/discussions)해보세요.
+관련된 질문은 [List-KR Discussions](https://github.com/List-KR/List-KR/discussions)에 남겨주세요.
 
 ---
 
 [![jsDelivr Stats](https://data.jsdelivr.com/v1/package/gh/List-KR/List-KR/badge)](https://www.jsdelivr.com/package/gh/List-KR/List-KR)
 
-**List-KR 필터는 [AdGuard](https://adguard.com) 광고 차단기와 [uBlock Origin](https://github.com/gorhill/uBlock)에만** 함께 사용되는 한국어 웹 사이트용 필터입니다.
+List-KR은 애드블록 커뮤니티와 AdGuard에서 관리하는 한국어 광고 차단 필터입니다.
 
-List-KR 필터는 광고 [^1]와 안티-애드블록을 필터하는 것을 목표로 합니다.
+[AdGuard](https://adguard.com)와 [uBlock Origin](https://github.com/gorhill/uBlock)에서 AdGuard 베이스 필터와 함께 사용했을 때 가장 잘 동작합니다.
```

Comment:
Enabling AdGuard Base filter means actually in AdGuard platform.
It seems that the previous description made a misunderstanding.

Reply:
```suggestion
공식적으로 지원하는 광고 차단기는 AdGuard와 uBlock Origin입니다.
AdGuard 플렛폼에서는 별도의 안내가 없는 경우 AdGuard 베이스 필터와 함께 사용하셔야 최상의 결과를 얻으실 수 있습니다.
```